### ISSUE TITLE
Consider Bintray is Cloased by JFrog in May

### ISSUE THREAD

**Issue by LeaderXinjianSun** [nielstiben/openTCS-NeNa#13]
Hi Niels,
Is the project influenced by Bintray,JCenter is Closed [Into the Sunset on May 1st: Bintray, GoCenter, and ChartCenter](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)? I just get an error like this `Could not resolve us.ihmc:euclid-geometry:0.15.1.`
<img src="https://user-images.githubusercontent.com/24226561/125554350-1ed632f1-3d6c-4063-b9c8-9b8ad64a730a.jpg" width="50%" height="50%">

**Comment by LeaderXinjianSun** [comment]
![屏幕截图 2021-07-14 154509](https://user-images.githubusercontent.com/24226561/125583890-3173027d-8784-418f-8fe3-05402c7492d4.jpg)
It seems like IHMCRobotics Repository is no longer be supported by Bintray.

**Comment by nielstiben** [comment]
Hello,

I'm aware of this issue. IHMCRobotics has indeed stopped its support for Bintray, as Bintray has ceased its services. This problem has been fixed on the development branch: https://github.com/nielstiben/openTCS-NeNa/tree/develop.

By the time I created this fix, IHMC was still working on transferring all libraries to the new platform (MavenCentral) . Therefore, I did not push the fix yet to a Release branch. I think that, nowadays IHMC is settled with MavenCentral, so I will push a new release with the fix somewhere next week.
